src/org/zaproxy/zap/extension/httpsinfo/ZapAddOn.xml

```diff
@@ -2,16 +2,19 @@
 	<name>HttpsInfo</name>
 	<version>7</version>
 	<status>alpha</status>
-	<description>Adds Https status information</description>
+	<description>Adds a context menu to the site tree and history table use of 
+	which displays HTTPS status information, scans for and 
+	alerts on some HTTPS vulnerabilities.</description>
```

Comment:
Using `<br />` doesn't seem to have any impact. Wrapping in CDATA tags and using `<br>` works but then makes the displayed description look bad. For this PR I'll revert to a short entry.